02-microarray/clustering/clustering_example.Rmd

```diff
@@ -1,20 +1,97 @@
-creating a heatmap.
-<br/> <i> Citation:</i> Gu Z, Eils R, Schlesner M (2016). “Complex heatmaps 
-reveal patterns and correlations in multidimensional genomic data.” Bioinformatics.
+Clustering can help you investigate which samples (or genes) are more related to each other.
+Here we use the Bioconductor R package `ComplexHeatmap` for clustering and creating a heatmap.
+In this example, the gene expression matrix is subset to only include genes with high variance.
+This set of genes is then used to create a heatmap.
+
+## Further learning resources about this analysis
+
+ - [Clustering strategies](https://www.statsandr.com/blog/clustering-analysis-k-means-and-hierarchical-clustering-by-hand-and-in-r/)  
+ - [Complex Heatmap book](https://jokergoo.github.io/ComplexHeatmap-reference/book/)  
+ - [Complex Heatmap vignette](https://bioconductor.riken.jp/packages/3.4/bioc/vignettes/ComplexHeatmap/inst/doc/s1.introduction.html)  
+ - [Complex Heatmap publication](https://academic.oup.com/bioinformatics/article/32/18/2847/1743594)  
+ 
+***
+
+# How to run this example
+
+## About dataset we are using for this example
+
+For this example analysis, we will use this [zebrafish microarray dataset](
+https://www.refine.bio/experiments/GSE12955/gene-expression-profile-of-zebrafish-kidney-side-population-sp-cells)
+This dataset has two experimental groups of lymphoid cells from zebrafish kidney.
+Using Hoechst fluorescence intensity, these were subdivided into the SP and MP populations. 
+
+## Obtaining the dataset from refinebio
+
+Go to this [dataset's page on refinebio](https://www.refine.bio/experiments/GSE12955/gene-expression-profile-of-zebrafish-kidney-side-population-sp-cells).  
+
+Click the "Download Now" button on the right side of this screen.  
+
+<img src="screenshots/download-now.png" width=200>  
+
+Fill out the pop up window with your email and our Terms and Conditions:  
+
+<img src="screenshots/download-email.png" width=500>  
+
+This will download a zip file that has a name with a series of letters and numbers and ends in `.zip`. 
+Double clicking should unzip this for you and create a folder of the same name.  
+<img src="screenshots/download-folder-structure.png" width=400> 
+
+For more details on the contents of this folder see [these docs on refinebio](http://docs.refine.bio/en/latest/main_text.html#rna-seq-sample-compendium-download-folder).  
+
+The `<experiment_accession_id>` folder has the data and metadata TSV files you will need for this example analysis.
+Experiment accession ids usually look something like `GSE1235` or `SRP12345`. 
+
+### Obtaining this example R notebook
+
+To run the examples yourself, you need to [download the Rmd by clicking this link](https://alexslemonade.github.io/refinebio-examples/02-microarray/clustering/clustering_example.Rmd). 
+Clicking this link will most likely send this to your downloads folder on your computer. 
+
+### Set up your analysis folder
+
```

Comment:
Can we give them the analysis folder when they download the rmd? We can still explain why it is set up the way it is .

Reply:
Maybe?? I don't know what is involved in including a folder set up with it. Will file an issue and look into that.

Reply:
Having not read the rest of the notebook (full disclosure), why not create a folder through normal means (e.g., `dir.create()`)?

Reply:
We still need the data files themselves to get in there. Right now we have them download the refinebio files and manually move it. We could do that but they'd have to stop running the analysis a few steps in (after they create the data folder) to move the refinebio files into this folder. I don't think we know for sure where they will be downloading the refinebio files to, so that's why it's manual like this. This is somewhat based on Josh's idea of trying to walk users through getting the dataset themselves as opposed to wgetting or etc.

Reply:
Related to #100